kibbeh/public/locales/nl/translation.json

```
@@ -119,9 +119,9 @@
       "starting": "Aan het downloaden...",
       "failed": "De automatische download is mislukt, probeer het later nog een keer",
       "visit_gh": "Bezoek Github Releases",
-      "prompt": "Click on the button below to start download",
+      "prompt": "Klik op de onderstaande knop om het downloaden te starten",
       "download_now": "Nu downloaden",
-      "download_for": "Download voor {{platform}}"
+      "download_for": "Download voor %platform%"
```

Why did you change that?

---

The placeholder wasn't working. It just said "Download voor", I looked in the code, and it used %, not {{}}. In the new deployed test version it works and says "Download voor Windows" etc.

---

@BronzW he did it correct, however @DoeHetGewoon  you're missing the `(%ext%)`
```suggestion
      "download_for": "Download voor %platform% (%ext%)"
```